controller/Publish.php

```diff
@@ -25,6 +25,8 @@
 use oat\taoDeliveryRdf\model\DeliveryFactory;
 use oat\taoDeliveryRdf\model\tasks\CompileDelivery;
 use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use oat\taoQtiItem\model\qti\Service as QtiItemService;
+use oat\taoQtiTest\models\TestModelService;
 use \tao_helpers_Uri;
```

Comment:
FQCN in `use` statements must not have a heading `\`.
(I know you didn't modify this @k-v-vladimir )